client/templates/Report/Report/_report_row.html.twig

```diff
@@ -14,9 +14,6 @@
         <p class="text flush--bottom govuk-body">
             {% block bottomLeftContent %}
             {% endblock %}
-            {#{{ bottomLeftContent | raw }}#}
-            {#{{ (page ~ '.editReportingPeriodPara') | trans(transOptions) }}#}
```

Comment:
Was this translation removed all together?

Reply:
It was already commented out so I just removed the commented out text

Reply:
Ah didn't spot that it was commented out.